dashboard-react.js

```diff
@@ -590,7 +590,82 @@ NETDATA.registryHello = function (host, callback) {
     }
   })
   .fail(function () {
-    NETDATA.error(407, host);
+    // NETDATA.error(407, host);
+
+    if (typeof callback === 'function') {
+      return callback(null);
+    }
+  });
+}
+
+NETDATA.registrySearch = function (machine_guid, getFromRegistry, serverDefault, callback) {
```

Comment:
those are copies from old registry code, we'll refactor this when we'll get rid of main.js